File: Modul 2/third.task.py
print('\n<<<Kegiatan 3>>>\n')

numbers = ((1, 2), (3, 4))

def square(listNumber):
    newListNumber = []
    for i in listNumber:
        for j in i:
            newListNumber.append(pow(j, 2))
    return tuple(newListNumber)

print(square(numbers))

#Robb - Mobile & Web Enthusiast

File: Modul 2/third-task.js
console.log('\n<<<Kegiatan 3>>>\n');

const numbers = [[1, 2], [3, 4]];

function square(listNumber) {
    const newListNumber = [];
    for (const i of listNumber) {
        for (const j of i) {
            newListNumber.push(Math.pow(j, 2));
        }
    }
    return newListNumber;
}

console.log(square(numbers));

